src/views/DocumentationTopic.vue

```diff
@@ -9,12 +9,57 @@
 -->
 
 <template>
-  <CodeTheme>
-    <Topic
-      v-if="topicData"
-      v-bind="topicProps"
-      :key="topicKey"
-    />
+  <CodeTheme class="doc-topic-view">
```

Comment:
I moved the sidebar navigator and the nav out into the view, because every page navigation was re-rendering the navigator, re-downloading the index, losing scrolling position, expanded status and so on. It was in overall not a great experience, especially for larger frameworks.

I could have moved fetching the index into the view, but we still had to deal with the reset of the navigator state.

I could not move just the navigator, as it actually lives below the nav, so I moved the nav too, which meant I had to move a few computed props too.